### PR TITLE
Update docs

### LINKED ISSUE #426: Incorrect documentation on edit group's endpoint
The example given as request body indicates parametter members as a list of string but it only works if a list of objects is given.

### PR DESCRIPTION
**Pull request by rorro**
I will go through them all but all of them might not need updating.

- [x] Achievements
- [x] Competitions
- [x] Deltas
- [x] Groups
- [x] Names
- [x] Players
- [x] Records
- [x] Snapshots

Closes #426

### CODE REVIEW

**Comment by rorro** [comment]
Things to take a second look at.

When removing a player from a group. Is this message possible or not.
```
{
  description: 'If none of the members given exist.',
  body: { message: 'No valid tracked players were given.' }
}
```

**Comment by psikoi** [comment]
Can you replace the base URL `https://wiseoldman.net/api` with `https://api.wiseoldman.net` in the API introduction page please?

**Comment by psikoi** [comment]
Also on that same page, the rate limit isn't 200 anymore, it is now 500

File: docs/src/configs/docs/names/endpoints.js [hunk 1]
```diff
@@ -60,23 +73,35 @@ export default [
...
       {
-        id: 12,
-        playerId: 13432,
-        status: 0,
-        oldName: 'zezima',
-        newName: 'lynx titan',
-        updatedAt: '2020-08-05T23:58:04.194Z',
-        createdAt: '2020-08-05T23:58:04.194Z',
-        resolvedAt: null
+        description: 'Note: Only showing top 3 for demo purposes',
```
Comment (psikoi) [inline docs/src/configs/docs/names/endpoints.js:76]: This note doesn't apply to this endpoint, could be a copy & paste mistake?

File: docs/src/configs/docs/records/entities.js [hunk 2]
```diff
@@ -42,6 +42,18 @@ export default [
       'All the possible values for the "period" field of the record model. (Note: a month is 31 days)',
     values: ['6h', 'day', 'week', 'month', 'year']
   },
+  {
+    name: 'Player types',
+    description:
+      'All the possible values for the "playerType" field of the record model.',
```
Comment (psikoi) [inline docs/src/configs/docs/records/entities.js:48]: There is no playerType in the record model, maybe change this to `...for the "playerType" query parameter.`

**The same applies for the playerBuild below.**

File: docs/src/configs/docs/deltas/entities.js [hunk 3]
```diff
@@ -442,6 +442,17 @@ export default [
       'All the possible values for the "period" field of the delta model. (Note: a month is 31 days)',
     values: ['6h', 'day', 'week', 'month', 'year']
   },
+  {
+    name: 'Player types',
+    description:
+      'All the possible values for the "playerType" field of the delta model.',
```
Comment (psikoi) [inline docs/src/configs/docs/deltas/entities.js:448]: There is no playerType in the delta model, maybe change this to ...for the "playerType" query parameter.

The same applies for the playerBuild below.

**Comment by psikoi** [comment]
`JsonBlock.jsx` line 5:

If you add `white-space: pre-wrap` to those styles, it wraps the json text and resolves that issue with the long error messages.

File: docs/src/configs/docs/records/entities.js (hunk 2 above)
Comment (rorro) [reply docs/src/configs/docs/records/entities.js]: Fixed in ae1a27c

File: docs/src/configs/docs/deltas/entities.js (hunk 3 above)
Comment (rorro) [reply docs/src/configs/docs/deltas/entities.js]: Fixed in ae1a27c

**Comment by psikoi** [comment]
`Participation` in competition/entities is missing two properties:

- startSnapshotId
- endSnapshotId

These are the first and last snapshots of a player's competition participation. The player's progress is the subtraction (endSnapshot - startSnapshot)

**Comment by psikoi** [comment]
In "Create Competition", there's the following comment:

> The response will contain a "verificationCode", this code must be stored as it is not possible to edit or delete the competition at a later date without it.

Maybe it could be edited to be

> If not a group competition, the response will contain a "verificationCode", this code must be stored as it is not possible to edit or delete the competition at a later date without it.

**Comment by psikoi** [comment]
On competition "update all":

> This action will perform a soft-update, meaning it won't import the player from CML or determine it's type.

This comment is outdated and no longer true

**Comment by psikoi** [comment]
On "GET/groups/:id/gained" it seems the success response has 1 too many levels of [ ]

**Comment by psikoi** [comment]
Everything else looks good to go! 👍

**Comment by rorro** [comment]
> On "GET/groups/:id/gained" it seems the success response has 1 too many levels of [ ]

**Comment by rorro** [comment]
I did not mean to close it. Oh well. The above comments have been fixed in 26b3abb

**Pull request merged**